.github/actions/update-rapidpro-workspace-settings/action.yml

```diff
@@ -0,0 +1,85 @@
+name: 'Update rapidpro workspace settings'
+inputs:
+  directory:
+    description: The path to the directory containing the configuration to be updated.
+    required: true
+    default: '.'
+
+  hostname:
+    description: The hostname of the instance to receive the configuration.
+    required: true
+
+  couch_node_name:
+    description: The name of your CouchDB's node.
+    required: true
+
+  username:
+    description: The username of an administrator account on the instance at "inputs.hostname". This user will make the deployment.
+    required: true
+
+  password:
```

Comment:
hostname, username & password can be reused from deploy-with-medic-conf

Reply:
Maybe, if the two jobs are used together.